### PR TITLE
Non stop tickets

### PR DESCRIPTION
**Pull request by Petsoliak**

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -1,17 +1,13 @@
...
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
 export class AppComponent implements OnInit {
 
-  constructor(private store: Store) {
-  }
-  ngOnInit() {
-    this.store.dispatch([new GetCountries(), new GetCities(), new GetAirports()]);
```
Comment (olena-dev) [inline src/app/app.component.ts:15]: revert this.

File: src/app/components/forTest/first/first.component.html [hunk 2]
```diff
@@ -1 +1,2 @@
 <p>first works!</p>
+<app-non-stop-tickets></app-non-stop-tickets>
```
Comment (olena-dev) [inline src/app/components/forTest/first/first.component.html:2]: add your component to the route and nav menu

File: src/app/components/non-stop-tickets/non-stop-tickets.component.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from '@angular/core';
+import { FlightsInfoService } from 'src/app/services/flights-info.service';
+import { Store } from '@ngxs/store';
+// import { GetAirports, GetCities, GetCountries } from 'src/app/store/request-data.action';
```
Comment (if137angular) [inline src/app/components/non-stop-tickets/non-stop-tickets.component.ts:4]: remove commented code

File: src/app/components/non-stop-tickets/non-stop-tickets.component.ts [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  selector: 'app-non-stop-tickets',
+  templateUrl: './non-stop-tickets.component.html',
+  styleUrls: ['./non-stop-tickets.component.scss']
+})
+export class NonStopTicketsComponent implements OnInit {
+
+  constructor(private store: Store, private flightsInfoService: FlightsInfoService) { }
+
+  ngOnInit(): void {
+    // this.store.dispatch([new GetCountries(), new GetAirports(), new GetCities()]);
```
Comment (if137angular) [inline src/app/components/non-stop-tickets/non-stop-tickets.component.ts:16]: remove commented code in all places

File: src/app/components/non-stop-tickets/non-stop-tickets.component.ts [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+    this.flightsInfoService.requestGetNonStopTickets().subscribe((res) => {
+      console.log(res);
```
Comment (if137angular) [inline src/app/components/non-stop-tickets/non-stop-tickets.component.ts:19]: remove console.log

File: src/app/app.component.ts [hunk 6]
```diff
@@ -1,17 +1,13 @@
...
+  constructor() {
```
Comment (if137angular) [inline src/app/app.component.ts:9]: revert store and dispatch as it was before

**Review by if137angular — CHANGES_REQUESTED** [review]

**Review by olena-dev — APPROVED** [review]

**Review by olena-dev — APPROVED** [review]

**Review by olena-dev — APPROVED** [review]

**Review by if137angular — APPROVED** [review]

**Pull request merged**